### PR TITLE
fix(`hana`): use keyword list from compiler

### PR DESCRIPTION
**Pull request by patricebender**
today we updated the keywords in the compiler because new hana version came with some new keywords. Instead of copying the keywords to this repo, we should rather import the list from the compiler to get rid of this double maintenance.

I already requested that also keywords for other sql dialects will be available in a similar fashion.

### CODE REVIEW

**Review by BobdenOs — APPROVED** [review]
👍 would be great for the others as well.

**Comment by patricebender** [comment]
> 👍 would be great for the others as well.

added this for sqlite in #525.
With the next compiler version there will be a similar API for the postgres keywords 💯

File: hana/lib/HANAService.js [hunk 1]
```diff
@@ -6,6 +6,12 @@ const { SQLService } = require('@cap-js/db-service')
 const drivers = require('./drivers')
 const cds = require('@sap/cds')
 const collations = require('./collations.json')
+const keywords = require('@sap/cds-compiler').to.hdi.keywords
```
Comment (johannes-vogel) [inline hana/lib/HANAService.js:9]: with that we have introduced a dependency to compiler at runtime. So far we did not have that. We either need to declare it or keep the local list. If this is all we need, I'd vote for a local list.

Comment (patricebender) [reply hana/lib/HANAService.js:9]: dont we have the dependency in any case due to the dependency to `sap/cds`?

(I think `cds` also exports the keywords, we could use the cds api instead)

Comment (johannes-vogel) [reply hana/lib/HANAService.js:9]: If cds exports it, that's fine.
I was just thinking that we introduce a dependency to the compiler only for keywords which seems like an overkill to me.

Comment (patricebender) [reply hana/lib/HANAService.js:9]: → #530

**Pull request merged**